Source/FieldSolver/FiniteDifferenceSolver/EvolveM.cpp

```diff
@@ -56,11 +56,12 @@ void FiniteDifferenceSolver::EvolveM (
         std::array< std::unique_ptr<amrex::MultiFab>, 3 > const& Bfield,
         amrex::Real const dt )
     {
-        static constexpr amrex::Real gamma = 1.759e-11; // gamma_L is gamma/(1+alpha^2), alpha^2 ~ 0
-        static constexpr amrex::Real alpha = 1e-4;
-        static constexpr amrex::Real Ms = 1e4;
-        Real constexpr cons1 = -gamma; // should be mu0*gamma, mu0 is absorbed by B used in this case
-        Real constexpr cons2 = -cons1*alpha/Ms; // factor of the second term in scalar LLG
+        static constexpr amrex::Real mag_alpha = 1e-4;
+        static constexpr amrex::Real Ms = 8e5;
+        static constexpr amrex::Real mag_gamma = 1.759e-11; // gyromagnetic ratio
+
+        Real constexpr cons1 = -mag_gamma; // should be mu0*mag_gamma, mu0 is absorbed by B used in this case
+        Real constexpr cons2 = -cons1*mag_alpha/Ms; // factor of the second term in scalar LLG
 
```

Comment:
Is EvolveM() ever supposed to be used, or only MacroscopicEvolveM()?
This function as is doesn't allow you to change the material properties without changing the source code.

Reply:
EvolveM() is not used for now. It is kept here just  so that in the future we are interested in applications like nuclear spin resonances which mostly doesn't involve varying macroscopic material properties. I'll fix EvolveM after this PR is approved. Or I can delete it for now if preferred.

Reply:
I suggest fixing it or adding an Abort() in there so nobody inadvertently runs it incorrectly.

Reply:
I added the amrex::Abort in the beginning of EvolveM.cpp

Reply:
I would add one more abort by these definitions of mag_alpha, etc., saying that the code should be rewritten so these can be specified in the inputs file (so we won't forget).

Reply:
I think I got your idea (and I totally agree that we should have some remark about the user-defined parameters), but I'm not quite sure if I got how you wanted it to be fixed. Could you clarify or we can have a quick meeting tomorrow?

Reply:
Maybe add something like: "amrex::Abort("Macroscopic parameters in EvolveM.cpp should be set via inputs file");"
Just don't let the code run and put an Abort stating it still needs to be fixed.  Up to you if you think this should be added.